src/badgehub-api-client/generated/swagger/private/private.ts

```diff
@@ -0,0 +1,344 @@
+/**
+ * Generated by orval v7.1.1 🍺
+ * Do not edit manually.
+ * badgehub-api
+ * Node project for the BadgeHub API
+ * OpenAPI spec version: 3
+ */
+import type {
+  DbInsertAppMetadataJSONPartial,
+  ProjectProps,
+  ProjectPropsPartial,
+  ProjectSlug,
+  Uint8Array,
+  UserProps,
+  Version,
+  WriteFileBody
+} from '../../models'
+
+/**
+ * Create a new app
+ */
+export type createAppResponse = {
+  data: void;
+  status: number;
+}
+
+export const getCreateAppUrl = (slug: ProjectSlug,) => {
+
+
+  return `http://localhost:8081/api/v3/apps/${slug}`
```

Comment:
Where are these references to `localhost` generated from?

Reply:
But that actually also brings me to the point that this should not be hardcoded in the generated files.
I'll check if I can change that.

Reply:
OK updated now: 
https://github.com/badgeteam/badgehub-frontend/pull/16/commits/c52220ff4af09bd8a10d12a129852a85c23a370e